measurement/validation.py

```diff
@@ -283,3 +297,26 @@ def save_validated_days(data: pd.DataFrame) -> None:
             variable__variable_code=row["variable"],
             time__date=row["date"],
         ).update(is_validated=True, is_active=not row["deactivate?"])
+
+
+def reset_validated_days(
+    station: str, variable: str, start_date: datetime.date, end_date: datetime.date
+) -> None:
+    """Resets validation and active status for the selected data.
+
+    TODO: should this also reset any modified value, minimum or maximum entries?
+
+    Args:
+        station (str): Station code
+        variable (str): Variable code
+        start_date (str): Start date
+        end_date (str): End date
+    """
+    Measurement.objects.filter(
+        station__station_code=station,
+        variable__variable_code=variable,
+        time__date__range=(
+            datetime.strptime(start_date, "%Y-%m-%d").date(),
+            datetime.strptime(end_date, "%Y-%m-%d").date(),
```

Comment:
I've just realised that these dates are naive - i.e. they don't have timezone attached. You will need to get the timezone from the station, first, and then construct a datetime object using that one. Otherwise, the entries that are considered within a particular day might be wrong.